flatnav/Index.h

```diff
@@ -615,6 +632,10 @@ template <typename dist_t, typename label_t> class Index {
                                  /* y = */ getNodeData(neighbor_node_id),
                                  /* asymmetric = */ true);
 
+      if (_collect_stats) {
+        _distance_computations.fetch_add(1);
```

Comment:
it would be more efficient to just do (out of the for-loop)
```c++
_distance_computations.fetch_add(_M); # _M being the number of edges
```
but we can't really do that because we don't know how many edges among the _M edges are not self-loops!